### ISSUE TITLE
Setting a language in the installer only sets `LC_CTYPE` and all the rest are set as `"POSIX"`

### ISSUE THREAD

**Issue by signed-log** [openSUSE/agama#289]
Host : Hyper-V generation 2 VM, 2vCores/4GB
Build : d-installer-live.x86_64-0.4.2-Build2.15.iso

Steps to reproduce :

- Select a language in the installer (taken French here)

Issue :

Running `locale` gives

```
LANG="POSIX"
LC_CTYPE=fr_FR.UTF-8
[...]
LC_ALL=
```

And the rest of `LC_*` being `"POSIX"`

Expected :

```
LANG=fr_FR.UTF-8
LC_CTYPE=fr_FR.UTF-8
[...]
LC_ALL=
```

And the rest of `LC_*` being `fr_FR.UTF-8`

**Comment by imobachgs** [comment]
Thanks for reporting. This one might be related to #274 (although, indeed, it is a different issue). We will plan for it.

**Comment by ancorgs** [comment]
This should be addressed by #881. If the issue persists next week (when an image containing those changes will be already released), please reopen.
